### PR TITLE
OpenVino - more fp16 tests

### PR DESCRIPTION
**Pull request by cnamrata15**

### CODE REVIEW

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/cum_sum.cpp [hunk 1]
```diff
@@ -11,8 +11,8 @@ using LayerTestsDefinitions::CumSumLayerTest;
 namespace {
 
 const std::vector<InferenceEngine::Precision> netPrecisions = {
-    InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::I32, InferenceEngine::Precision::FP32,
+    // InferenceEngine::Precision::FP16,
```
Comment (tzerrell) [inline plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/cum_sum.cpp:15]: I think moving the commented-out option to the beginning of the list will clean up the linting (to enable one-per-line).

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/extract_image_patches.cpp [hunk 2]
```diff
@@ -11,8 +11,8 @@ using LayerTestsDefinitions::ExtractImagePatchesTest;
 namespace {
 
 const std::vector<InferenceEngine::Precision> netPrecisions = {
-    InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::I32, InferenceEngine::Precision::FP32,
+    // InferenceEngine::Precision::FP16,
```
Comment (tzerrell) [inline plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/extract_image_patches.cpp:15]: Move to top (see prev. comment)

**Review by tzerrell — COMMENTED** [review]
Looks good in general; I have a couple of linting comments.

Also we'll need to track the commented-out cases but I think it makes sense to land before pursuing fixes of those since that way we'll at least be testing things that are currently working.

**Review by tzerrell — DISMISSED** [review]

**Review by tzerrell — APPROVED** [review]

**Comment by flaub** [comment]
@tzerrell @cnamrata15 what's the status of this PR?

**Comment by tzerrell** [comment]
Correction: After investigating, the changes from here were either copied to other PRs or adapted into other PRs; none of them are outstanding. Closing this PR as obsolete.

**Pull request closed without merging**